backend/main.go

```diff
@@ -29,7 +30,13 @@ func main() {
 	log.Printf("Server started")
 	log.Printf("Authentication method: %s", singleton.GetAuthenticationMethod())
 	router := sw.NewRouter()
+
+	corsHandler := handlers.CORS(
+		handlers.AllowedOrigins([]string{"*"}), // Otwiera na wszystkie domeny
```

Comment:
This comment needs to be removed.

Reply:
I left it on purpose, because eventually it will have to be changed to some kind of traffic narrowing, and not be open to all addresses all the time. I'm going to change it to emphasize this even more and not to be in Polish

Reply:
Ok, I just thought it was comment from chatgpt that got lost.